dev/TeachingTip/InteractionTests/TeachingTipTests.cs

```diff
@@ -502,6 +504,183 @@ public void SpecifiedPlacement()
             }
         }
 
+        [TestMethod]
+        public void SpecifiedPlacementRTL()
+        {
+            using (var setup = new TestSetupHelper("TeachingTip Tests"))
+            {
+                elements = new TeachingTipTestPageElements();
+
+                foreach (TipLocationOptions location in Enum.GetValues(typeof(TipLocationOptions)))
+                {
+                    SetTeachingTipLocation(location);
+
+                    ScrollTargetIntoView();
+                    ScrollBy(10);
+
+                    new CheckBox(FindElement.ByName("PageRTLCheckbox")).Check();
```

Comment:
>FindElement [](start = 33, length = 11)

Teaching Tip's test page uses the TeachingTipTestPageElements class to get things like this, I'd prefer that this get moved there.

Reply:
#Fixed